### ISSUE TITLE
Project required including beam-runners-core-java

### ISSUE THREAD

**Issue by MikaelSmith** [ngrunwald/datasplash#61]
To get my project working, I had to explicitly include `[org.apache.beam/beam-runners-core-java "2.8.0"]` after upgrading from datasplash 0.5.3 to 0.6.3.

My project looks like
```
(ns my-pipeline.core
  (:gen-class)
  (:require [cheshire.core :as json]
            [clojure.pprint :as pp]
            [clojure.tools.logging :as log]
            [datasplash.api :as datasplash]
            [datasplash.bq :as bq]
            [datasplash.core :as core]
            [datasplash.pubsub :as pubsub]))

# Actual work pulled out to keep it compact
(defn clean-data [m] m)

(defn log-data [m]
  (let [out (java.io.StringWriter.)]
    (pp/pprint m out)
    (log/info (.toString out)))
  m)

(datasplash/defoptions Options
  {:pubsubTopic {:type String
                 :description "Input PubSub topic"}
   :bigqueryTable {:type String
                   :description "Output BigQuery table as PROJECT:DATASET.TABLE"}
   :tempLocation {:type String}})

(defn -main
  [& args]
  (let [pipeline (datasplash/make-pipeline Options args {:streaming true})
        {:keys [pubsubTopic bigqueryTable]} (datasplash/get-pipeline-options pipeline)]
    (->> pipeline
         (pubsub/read-from-pubsub pubsubTopic {:kind :topic})
         (datasplash/map #(json/decode % true))
         (datasplash/map clean-data)
         (datasplash/map log-data)
         (bq/write-bq-table bigqueryTable {:create-disposition :never
                                           :write-disposition :append
                                           :retry-policy :retry-transient})
         (datasplash/run-pipeline)))
```

**Comment by m-lce** [comment]
Hello,
Coudl you give us more details about the error you have?
Thank you

**Comment by MikaelSmith** [comment]
```
Exception in thread "main" java.lang.reflect.InvocationTargetException, compiling:(/private/var/folders/gr/lnzxwt5j0736c6rwk6yjq92h0000gn/T/form-init2301896090944576632.clj:1:125)
	at clojure.lang.Compiler.load(Compiler.java:7526)
	at clojure.lang.Compiler.loadFile(Compiler.java:7452)
	at clojure.main$load_script.invokeStatic(main.clj:278)
	at clojure.main$init_opt.invokeStatic(main.clj:280)
	at clojure.main$init_opt.invoke(main.clj:280)
	at clojure.main$initialize.invokeStatic(main.clj:311)
	at clojure.main$null_opt.invokeStatic(main.clj:345)
	at clojure.main$null_opt.invoke(main.clj:342)
	at clojure.main$main.invokeStatic(main.clj:424)
	at clojure.main$main.doInvoke(main.clj:387)
	at clojure.lang.RestFn.applyTo(RestFn.java:137)
	at clojure.lang.Var.applyTo(Var.java:702)
	at clojure.main.main(main.java:37)
Caused by: java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at clojure.core$bean$fn__6906$fn__6907.invoke(core_proxy.clj:408)
	at clojure.core$bean$snapshot__6913$fn__6914.invoke(core_proxy.clj:417)
	at clojure.core$reduce1.invokeStatic(core.clj:936)
	at clojure.core$bean$snapshot__6913.invoke(core_proxy.clj:415)
	at clojure.core$bean$fn__6936.invoke(core_proxy.clj:434)
	at clojure.core.proxy$clojure.lang.APersistentMap$ff19274a.without(Unknown Source)
	at clojure.lang.RT.dissoc(RT.java:884)
	at clojure.core$dissoc.invokeStatic(core.clj:1503)
	at clojure.core$dissoc.invoke(core.clj:1496)
	at datasplash.core$get_pipeline_options.invokeStatic(core.clj:1034)
	at datasplash.core$get_pipeline_options.invoke(core.clj:1027)
	at dujour_pipeline.core$pipeline.invokeStatic(core.clj:63)
	at dujour_pipeline.core$pipeline.invoke(core.clj:60)
	at dujour_pipeline.core$_main.invokeStatic(core.clj:78)
	at dujour_pipeline.core$_main.doInvoke(core.clj:75)
	at clojure.lang.RestFn.invoke(RestFn.java:457)
	at clojure.lang.Var.invoke(Var.java:396)
	at user$eval149.invokeStatic(form-init2301896090944576632.clj:1)
	at user$eval149.invoke(form-init2301896090944576632.clj:1)
	at clojure.lang.Compiler.eval(Compiler.java:7062)
	at clojure.lang.Compiler.eval(Compiler.java:7052)
	at clojure.lang.Compiler.load(Compiler.java:7514)
	... 12 more
Caused by: java.lang.IllegalArgumentException: NoOpMetricsSink was not found on classpath
	at org.apache.beam.sdk.options.PipelineOptions$NoOpMetricsSink.create(PipelineOptions.java:433)
	at org.apache.beam.sdk.options.PipelineOptions$NoOpMetricsSink.create(PipelineOptions.java:419)
	at org.apache.beam.sdk.options.ProxyInvocationHandler.returnDefaultHelper(ProxyInvocationHandler.java:592)
	at org.apache.beam.sdk.options.ProxyInvocationHandler.getDefault(ProxyInvocationHandler.java:533)
	at org.apache.beam.sdk.options.ProxyInvocationHandler.invoke(ProxyInvocationHandler.java:158)
	at com.sun.proxy.$Proxy46.getMetricsSink(Unknown Source)
	... 38 more
```

**Comment by RolT** [comment]
This was solved with ff998fa06e82c6620eb34d3ce69d21bf9bb05a2e, released in version `0.6.4`
